Source/Projections/Builder/UnregisteredProjections.cs

```diff
@@ -28,7 +31,11 @@ public UnregisteredProjections(IUniqueBindings<ProjectionModelId, IProjection> p
         : base(projections)
     {
         ReadModelTypes = readModelTypes;
+        AddTenantScopedServices = AddToContainer;
```

Comment:
Hmm, I'm wondering here why you set this property to a private method (kind of like a callback), instead of having that method on the interface to be implemented directly? Is there some clever reasoning here that I don't see?

Reply:
No particular reason, I just thought it looked nice and familiar. I'm open to suggestions